src/appimagetool.c

```diff
@@ -890,10 +914,27 @@ main (int argc, char *argv[])
             exit(1);
         }
         
-        /* If the user has not provided update information but we know this is a Travis CI build,
-         * then fill in update information based on TRAVIS_REPO_SLUG */
+        /* If the user has not provided update information but we know this is a CI build,
+         * then fill in update information based on well-known CI environment variables */
         if(guess_update_information){
-            if(travis_repo_slug){
+
+            if(github_repository_name){
+                if(!github_token) {
+                    printf("Will not guess update information since $GITHUB_TOKEN is missing\n");
+                } else {
+                    gchar *zsyncmake_path = g_find_program_in_path ("zsyncmake");
+                    if(zsyncmake_path){
+                        char buf[1024];
+                        // gh-releases-zsync|probono|AppImages|latest|Subsurface-*x86_64.AppImage.zsync
+                        sprintf(buf, "gh-releases-zsync|%s|%s|latest|%s*-%s.AppImage.zsync", github_repository_owner, github_repository_name, app_name_for_filename, arch);
```

Comment:
Another `sprintf`.

Reply:
See [caa7f0d](https://github.com/AppImage/appimagetool/pull/13/commits/caa7f0d6327ef65e2fda8f33d9bcfe7060a01409).

Not sure whether I did too much of it now. Please check carefully whether my changes make sense. Thanks!

Reply:
GitHub Actions are red. I guess this was too much indeed. (Even though it seemed to compile locally.)